client/pages/index.tsx

```diff
@@ -1,151 +1,25 @@
-    socket.on('err_msg', onErrorMessage);
-
-    return () => {
-      console.log('Unmounting Component');
-
-      // clean up socket event listeners
-      socket.off('connect', onConnect);
-      socket.off('api_response', onApiResonse);
-      socket.off('send_photo', onSendPhoto);
-
-      if (intervalRef.current) {
-        clearInterval(intervalRef.current);
-      }
-    };
-  }, []); // run once on init
-
-  useEffect(() => {
-    // runs every time displayState changes
-    // console.log('display changed', display);
-  }, [display]);
-
-  useEffect(() => {
-    if (likesComplete && commentsComplete) {
-      console.log('Likes and comments animation completed');
-      setLikesComplete(false);
-      setCommentsComplete(false);
-      setImg('');
-      donePhoto();
-    }
-    // fire useEffect when either state vars change
-  }, [likesComplete, commentsComplete]);
-
-  const takePhoto = () => {
-    console.log('Taking a photo...');
-    socket.emit('take_photo');
-  };
-
-  const donePhoto = () => {
-    console.log('Completed displaying all comments and likes');
-    console.log('Clearing display');
-    setDisplay();
-    setIsLoading(false);
-
-    setTimeout(() => {
-      console.log('5s passed taking a new photo');
-      // TODO: declare this as a const, e.g., WAIT_TIME or better name
-      // time we want to wait/stall after the last photo is taken
-
-      setIsLoading(true);
-    }, 5000);
-  };
-
-  const onLikesComplete = () => {
-    setLikesComplete(true);
-  };
-
-  const onCommentsComplete = () => {
-    setCommentsComplete(true);
-  };
-
   return (
-    <>
-      <div className="w-screen h-screen flex flex-col justify-start items-center">
-        {display ? (
-          <>
-          <div className='w-screen flex flex-row justify-between align-start'>
-          <Likes finalLikes={display.likes} onComplete={onLikesComplete}></Likes>
-          {img && !(likesComplete && commentsComplete) && <Photo img={img} />}
-          </div>
-            
-            <CommentFeed comments={display.comments} onComplete={onCommentsComplete}></CommentFeed>
-          </>
-        ) : (
-          <>
-            {isLoading ? (
-              <>
-                <LoadingOverlay takePhoto={takePhoto}></LoadingOverlay>
-              </>
-            ) : (
-              <EndingOverlay></EndingOverlay>
-            )}
-          </>
-        )}
-        
-      </div>
-    </>
+    <div className="w-screen h-screen overflow-hidden bg-black relative">
+      <CameraFeed /> {/* This renders the live camera feed as background */}
+      <button
+        onClick={() => console.log("Button clicked")}
+        className="
+          bg-white/30 text-white text-3xl font-bold 
+          py-4 px-10 rounded-full opacity-90 
+          hover:opacity-100 transition-opacity 
+          active:bg-white/50 active:scale-95
+          absolute bottom-[25%] left-1/2 transform -translate-x-1/2 touch-auto cursor-pointer
```

Comment:
could you add the class `select-none` as well?

sometimes if i tap and hold for too long, i end up selecting the text, select-none (`user-select: none`) makes sure the text won't highlight in case the user is touching the screen for too long

e.g., 
![image](https://github.com/user-attachments/assets/60822bab-5d54-4bd5-81de-8e5629c567f2)

Reply:
sounds good and changes made! ty!